dynamic_programming/EditDistance.md

```diff
@@ -0,0 +1,271 @@
+# Editing Distance
+
+**Translator: [Master-cai](https://github.com/Master-cai)**
+
+**Author: [labuladong](https://github.com/labuladong)**
+
+few days ago, I saw an interview paper of Tencent. In this paper, most of the algorithm problems are Dynamic programming. The last question is that writing a function to calculate the shortest editing Distance. Today I wrote an article specifically to discuss this problem.
+
+I personally like this problem because it looks very hard, the solution is Surprisingly simple and beautiful and it`s a rare algorithm which is not very useful.(yech, I recognized that many algorithm problems are not very useful.)Following is the problem:
```

Comment:
```suggestion
I personally like this problem because it looks very hard, the solution is Surprisingly simple and beautiful and it\`s a rare algorithm which is not very useful. (yech, I recognized that many algorithm problems are not very useful.)Following is the problem:
```

It's 要用单引号，不能用反引号(用反引号需要打反斜杠 \ 转移），因为反引号在 markdown 语法中有特殊含义，会导致排版混乱，请修改